components/Link.tsx

```diff
@@ -4,13 +4,13 @@ import getConfig from 'next/config';
 
 const { publicRuntimeConfig } = getConfig();
 
-const Link: React.FunctionComponent<LinkProps> = ({ children, ...props }) => (
-  <NextLink
+const Link: React.FunctionComponent<LinkProps> = ({ children, ...props }) => {
+  return (<NextLink
```

Comment:
Hmm, is this better?

Reply:
ahh no just some debugging that I forgot to fully revert, fixed in ba32008f3b69336d83c8f207795f95c52796f753